app/views/layouts/_topbar.html.erb

```diff
@@ -1,4 +1,11 @@
-        <ul class="loginbar pull-right">
+<ul class="loginbar pull-right">
+  <%if user_signed_in? && current_user.admin?%>
+    <% commit = `git rev-parse HEAD`[0,6] %>
```

Comment:
I think that we should implement some kind of admin-controller that could be used for things like this. Until then, the menu controller seems like a sane choice, or leave it in the view (dirty).